python/lsst/analysis/drp/plotUtils.py

```diff
@@ -135,14 +142,25 @@ def addPlotInfo(fig, plotInfo):
     photocalibDataset = "None"
     astroDataset = "None"
 
-    plt.text(0.02, 0.98, "Run:" + plotInfo["run"], fontsize=8, alpha=0.8, transform=fig.transFigure)
-    datasetType = ("Datasets used: photocalib: " + photocalibDataset + ", astrometry: " + astroDataset)
-    tableType = "Table: " + plotInfo["tractTableType"]
-    dataIdText = "Tract: " + str(plotInfo["tract"]) + " , Filter: " + plotInfo["filter"]
-    plt.text(0.02, 0.95, datasetType, fontsize=8, alpha=0.8, transform=fig.transFigure)
+    plt.text(0.01, 0.99, plotInfo["plotName"], fontsize=8, transform=fig.transFigure, ha="left", va="top")
+
+    run = plotInfo['run']
+    datasetsUsed = f"\nPhotoCalib: {photocalibDataset}, Astrometry: {astroDataset}"
+    tableType = f"\nTable: {plotInfo['tractTableType']}"
+
+    dataIdText = ""
+    if str(plotInfo["tract"]) != "N/A":
+        dataIdText += f"Tract: {plotInfo['tract']}"
```

Comment:
The way you have this set up, you need to change this line to:
```
dataIdText += f", Tract: {plotInfo['tract']}"
```
Otherwise, the printed info block doesn't have a comma-space between the table and the `Tract:`.

That said, wouldn't it be cleaner to add your required commas and whitespaces directly into the `infoText` constructor below instead?